packages/client/vite.config.ts

```diff
@@ -10,32 +10,41 @@ export default defineConfig({
       strict: false,
     },
   },
-  // root: "src",
-  // build: {
-  //   outDir: "../dist",
-  //   emptyOutDir: true,
-  //   sourcemap: true,
-  //   assetsInlineLimit: 0,
-  //   target: "es2022",
-  // },
-  // preview: {
-  //   port: 3000,
-  // },
-  // resolve: {
-  //   dedupe: ["proxy-deep", "styled-components"],
-  // },
+  preview: {
+    port: 3000,
+  },
+  build: {
+    outDir: "../dist",
+    emptyOutDir: true,
+    // If you get out-of-memory issues, it's probably due to sourcemaps. Try
+    // doing more code splitting more manual chunks below.
+    sourcemap: true,
```

Comment:
this is causing OOM in CI, may need to disable there or figure out how to bump memory needs